yb-voyager/cmd/importData.go

```diff
@@ -896,6 +902,45 @@ func createFileTaskImporter(task *ImportFileTask, state *ImportDataState, batchI
 	return taskImporter, nil
 }
 
+func startMonitoringTargetHealth() error {
+	if !slices.Contains([]string{TARGET_DB_IMPORTER_ROLE, IMPORT_FILE_ROLE}, importerRole) {
+		return nil
+	}
+	if skipNodeHealthChecks && skipDiskUsageHealthChecks && skipReplicationChecks {
+		return nil
+	}
+	yb, ok := tdb.(*tgtdb.TargetYugabyteDB)
+	if !ok {
+		return fmt.Errorf("monitoring health is only supported if target DB is YugabyteDB")
+	}
+	go func() {
+		err := monitor.MonitorTargetHealth(yb, metaDB, skipDiskUsageHealthChecks, skipReplicationChecks, skipNodeHealthChecks, func(info string) {
```

Comment:
do we still need metadb, now that you are passing the callback function?

Reply:
yeah no, will remove it.